about.html

```diff
@@ -34,114 +34,25 @@ <h2>Project Leads</h2>
-      <p>Jason Paryani is a software engineer irresistibly drawn to systems topics. This passion led him to Philadelphia-area finance firm Susquehanna International Group, LLP. While at SIG, he worked on the backtesting team where he worked at the intersection of a .Net Windows stack meeting a Python managed Linux high performance cluster. In 2013, he moved to Mountain View to co-found Statsketch, a software startup focused on delivering easy to use data visualization tools for the web. He got to know Kenton when he implemented pycapnp, the Python port of Cap’n Proto.
-      <h5>Author, <a href="https://github.com/jparyani/pycapnp">pycapnp</a><br>Now working at <a href="https://cloudflare.com">Cloudflare</a></h5>
-
-    <li class="david">
-      <h3>David Renshaw</h3>
-      <h4>Programmer (emeritus)</h4>
-      <p>David Renshaw is a mathematician turned programmer. As an undergrad at Caltech, he published research in combinatorics, and as a grad student at Carnegie Mellon he developed techniques for building machine-verified proofs of safety properties for aircraft controllers and surgical robots. Some of his side projects have included a framework for crowdsourcing playthroughs of javascript games, and contributions to a Standard ML port of the Box2D physics engine. Prior to joining Sandstorm, he spent two years hunting for vulnerabilities in robotics software at the National Robotics Engineering Center. David is leading development on the Rust and Java implementations of Cap'n Proto.
-      <h5>Author, <a href="https://github.com/dwrensha/capnproto-rust">capnproto-rust</a><br>Author, <a href="https://github.com/dwrensha/capnproto-java">capnproto-java</a><br>Now working at <a href="https://aurora.tech/">Aurora Tech</a></h5>
+<section id="history">
+  <h2>Project History</h2>
+<p>Kenton Varda launched Sandstorm in 2014 via an <a href="http://igg.me/at/sandstorm">Indiegogo campaign</a>, raising $58,929 from 704 backers. With a $1.3M seed funding round, Kenton co-founded Sandstorm Development Group with <a href="https://github.com/jadeqwang">Jade Wang</a> to develop Sandstorm as both a Software-as-a-Service offering for consumers, known as Sandstorm Oasis, and an enterprise offering for businesses, known as Sandstorm for Work. SDG operated from 2014 until 2017, and employed <a href="https://github.com/jparyani">Jason Paryani</a>, <a href="https://github.com/dwrensha/">David Renshaw</a>, <a href="https://github.com/paulproteus">Asheesh Laroia</a>, <a href="https://github.com/neynah">Nena Nguyen</a>, and <a href="https://github.com/zarvox">Drew Fisher</a>, significantly expanding both Sandstorm itself, its collection of applications and developer tools such as vagrant-spk, and developing Blackrock, a scalable version of Sandstorm to back Oasis. SDG was also advised by operating system and web security experts including Brian Swetland, Andrew Lutomirski, Jasvir Nagra, Mark Seaborn, and Mark S. Miller.</p>

```

Comment:
Minor thoughts on this: I was looking here for nods to the team that originally was part of SDG. Glad to see it here, but on first read "employed" feels slightly off. Additionally, that sentence is slightly run-on. One alternative that comes to my mind:

> `...known as Sandstorm for Work. The SDG team included <a href="https://github.com/jparyani">Jason Paryani</a>, <a href="https://github.com/dwrensha/">David Renshaw</a>, <a href="https://github.com/paulproteus">Asheesh Laroia</a>, <a href="https://github.com/neynah">Nena Nguyen</a>, and <a href="https://github.com/zarvox">Drew Fisher</a>. SDG operated from 2014 until 2017, significantly expanding both Sandstorm itself, its collection of applications and developer tools such as vagrant-spk, and developing Blackrock, a scalable version of Sandstorm to back Oasis. SDG was also advised...`